### DISCUSSION TITLE
„All“ Filter for PRs in repo view not saved (regression)

### DISCUSSION THREAD

**Discussion by tooomm** [community/community#49390] [Mobile]
When going to home and selecting a certain repo and browse PRs, there is a filter on status open/closed etc.
If one sets this to all to not only see open PRs, it correctly shows all PRs. Going back to repo overview and opening the PR section again resets the filter to open.

That was not the case ~2 releases before.
The app still remembers the custom view/filter for issues currently.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.103.0 (93212509)
Device: iPhone SE 2 (iOS 16.3.1)
</details>

**Comment by eliperkins** [comment]
Thanks for submitting this feedback, @tooomm! It does seem like this is a bug. I've filed an issue on our internal tracking repo for this. I'll follow back up with you when the bug has been fixed.
